src/commands/kv/list_keys.rs

```diff
@@ -0,0 +1,95 @@
+extern crate serde_json;
+
+use cloudflare::endpoints::workerskv::list_namespace_keys::ListNamespaceKeys;
+use cloudflare::endpoints::workerskv::list_namespace_keys::ListNamespaceKeysParams;
+use cloudflare::endpoints::workerskv::Key;
+use cloudflare::framework::apiclient::ApiClient;
+use failure::bail;
+use serde_json::value::Value as JsonValue;
+
+// Note: this function only prints keys in json form, given that
+// the number of entries in each json blob is variable (so csv and tsv
+// representation won't make sense)
+pub fn list_keys(id: &str, prefix: Option<&str>) -> Result<(), failure::Error> {
+    let client = super::api_client()?;
+    let account_id = super::account_id()?;
+
+    let params = ListNamespaceKeysParams {
+        limit: None, // Defaults to 1000 (the maximum)
+        cursor: None,
+        prefix: prefix.map(str::to_string),
+    };
+
+    let mut request_params = ListNamespaceKeys {
+        account_identifier: &account_id,
+        namespace_identifier: id,
+        params: params,
+    };
+
+    let mut response = client.request(&request_params);
+
+    print!("["); // Open json list bracket
+
+    // used to track whether to put a glue "," before printing a json blob.
+    // Iterate over all pages until no pages of keys are left.
+    // This is detected when a returned cursor is an empty string.
+    // todo(gabbi): the code in this loop is the product of a looooong fight
```

Comment:
can remove this comment :) i'm sure if folks find a reason to change it they will.

Reply:
will do!